index.html

```diff
@@ -1043,9 +1042,8 @@ <h4>
       we rely on async/await.
     </p>
     <pre class="example">
-      const ndef = new NDEFReader();
       try {
-        ndef.write({
+        navigator.nfc.ndef.write({
```

Comment:
Use `await` to catch exception.
```suggestion
        await navigator.nfc.ndef.write({
```